### PR TITLE
Rename "arb one" gov action styff "l2" gov actions

### PR DESCRIPTION
**Pull request by DZGoldman**
more general name b/c of testnet(s)

### CODE REVIEW

File: src/gov-action-contracts/address-registries/L2GovAddressRegistryInterfaces.sol [hunk 1]
```diff
@@ -64,7 +64,7 @@ interface IL2ArbitrumTokenGetter {
     function l2ArbitrumToken() external view returns (IL2ArbitrumToken);
 }
 
-interface IArbOneGovAddressRegistry is
```
Comment (yahgwai) [inline src/gov-action-contracts/address-registries/L2GovAddressRegistryInterfaces.sol:67]: What about Nova? We may need an address registry there at some point, one reason might be to handle DAC reward changes. What will call that address reg?

Comment (DZGoldman) [reply src/gov-action-contracts/address-registries/L2GovAddressRegistryInterfaces.sol:67]: I think keeping this one for gov-specific addresses and introducing a new registry for anytrust-specific things makes most sense

Comment (DZGoldman) [reply src/gov-action-contracts/address-registries/L2GovAddressRegistryInterfaces.sol:67]: ...then again, maybe just L1 vs. L2 address registry is nicest?

Comment (DZGoldman) [reply src/gov-action-contracts/address-registries/L2GovAddressRegistryInterfaces.sol]: you convinced me

**Review by gvladika — APPROVED** [review]
lgtm

**Pull request merged**